components/form/LabeledSelect.vue

```diff
@@ -334,6 +335,7 @@ export default {
 <style lang='scss' scoped>
 .labeled-select {
   position: relative;
+  max-height: 61px;
```

Comment:
stop select from expanding to the height of the name input including the validation error height